src/main/java/com/google/cloud/tools/appengine/AppEngineDescriptor.java

```diff
@@ -16,16 +16,23 @@
 
 package com.google.cloud.tools.appengine;
```

Comment:
There is: `src/test/java/com/google/cloud/tools/appengine/AppEngineDescriptorTest.java`